### DISCUSSION TITLE
Need for a code dictionary to check all accessible functions.

### DISCUSSION THREAD

**Discussion by DilemmaGX** [Gandi-IDE/custom-extension#124] [Q&A] [answered]
# Need for a code dictionary
Basically I'm willing to build an animation extension (**`Animator`**) but there's no indication (or maybe I didn't found it) about move_to(x,y) in the files.
~~_And I didn't find any scratch related commands in the example files_~~
You see, TurboWarp has one really general page to introduce functions to developers, and I think we all need one for Gandi.
It's going to help a lot if we make a webpage like that awa.

**Comment by FurryR** [comment, marked as answer]
Hello there! I'm FurryR (Shinogi Kumagai), an unofficial extension developer. I'm here to help you.

# What happened

As you see, Gandi has poor documentation. There's no code dictionary, so you have to read the source code to find out how to code.

Kinda wild, but you can see [scratch3_motion.js](https://github.com/TurboWarp/scratch-vm/blob/develop/src/blocks/scratch3_motion.js) for more information about your problem.

# How

Kinda wild, but you can see [scratch3_motion.js](https://github.com/TurboWarp/scratch-vm/blob/develop/src/blocks/scratch3_motion.js) for more information about your problem.

You can use `util.target` to get current target.

```js
util.target.setXY(0, 0);
```

For animations, see [`glide()`](https://github.com/TurboWarp/scratch-vm/blob/develop/src/blocks/scratch3_motion.js#L147).

You can use `util.yield()` to yield the execution.

---

*In order to publish your extension, you need to invite a **translator** to translate your extension into **Simplified Chinese**.*
